packages/volto-slate/src/blocks/Text/keyboard/slashMenu.js

```diff
@@ -1,5 +1,6 @@
 export const slashMenu = ({ editor, event }) => {
   if (!editor.showSlashMenu) return;
+  event.preventDefault();
```

Comment:
@danalvrz This may be called for any key press, so we should only prevent the default action if one of the handlers is used here!

So I would replace line 14 with this:
```
if (handler) {
    event.preventDefault();
    handler();
}
```